src/examples/install-aws-cli-with-web-identity.yml

```diff
@@ -0,0 +1,23 @@
+description: A usage expample that installs and configures a profile with Web Identity in the AWS CLI.
```

Comment:
> A usage expample

Spelling + No need to mention this is a usage example in the description, users will already know this is a usage example. 

Maybe something like:

> Setup the AWS CLI and configure with Web Identity. Assume roles on AWS without storing keys on CircleCI and utilize short-term credentials instead.

And we may again want to consider linking to docs here.